Dockerfile

```diff
@@ -1,13 +1,20 @@
 FROM opensuse/leap:15.0
 
-RUN zypper refresh
-RUN zypper --non-interactive dup
-RUN zypper --non-interactive install git
-RUN zypper --non-interactive install syslinux
-RUN zypper --non-interactive install xorriso
-RUN zypper --non-interactive install gfxboot
-RUN zypper --non-interactive install python3-kiwi
-RUN zypper --non-interactive install checkmedia
+RUN set -o errexit -o nounset -o xtrace \
+  ; zypper --non-interactive refresh \
+  ; zypper --non-interactive dup \
+  ; zypper --non-interactive install \
+    checkmedia \
+    gfxboot \
+    git-core \
+    patch \
+    python3-kiwi \
+    syslinux \
+    xorriso \
```

Comment:
This is mostly a cleanup (each `RUN` produces a new layer, which is more data to transfer for no reason); additionally, `git-core` is much smaller than the `git` metapackage.

Reply:
Perfect.